recipe/meta.yaml

```
@@ -1,34 +1,90 @@
+# Notes: this receipt is based on torchvision-feedstock: https://github.com/conda-forge/torchvision-feedstock/blob/master/recipe/meta.yaml
+
 {% set name = "torchani" %}
-{% set version = "2.2" %}
+{% set version = "2.2.1" %}
+{% set torchani_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: c086fe6cabe49a0408d1f1b12aaf14e33b9e6ef0ffb15f3020792934e7c9fbbe
-  
+  git_url: https://github.com/aiqm/{{ name }}.git
+  git_rev: {{ version }}
```

```suggestion
  url: https://github.com/aiqm/torchani/archive/refs/tags/{{ version }}.tar.gz
  sha256: c4f299a586518c57b17332309f803e6ae34c1b7a4b48d542bd6c46e9c4f8d146
```

We don't need published releases, tags are enough to have downloadable sources from github.

---

Hi, the downloaded tar.gz does not contain `.git` folder, which is needed for `setuptools-scm`, so tag could not work in this case. But let me know if you have better suggestions!

---

> But let me know if you have better suggestions!

Conda is able to take care of this: https://conda-forge.org/docs/maintainer/knowledge_base.html#using-setuptools-scm 🙃

---

Thanks! This is a good way!